kratos/python_scripts/assign_scalar_variable_process.py

```diff
@@ -39,7 +50,7 @@ def __init__(self, Model, settings ):
             "variable_name"   : "SPECIFY_VARIABLE_NAME",
             "interval"        : [0.0, 1e30],
             "constrained"     : true,
-            "value"           : 0.0,
+            "value"           : {},
```

Comment:
Why changing the defaults? Note that this would break any case that is assuming the zero scalar default value (admittedly a weird usage).

Reply:
OK. Now I see below. Forget about my comment.

Reply:
I think you're right, if a user does not set `"value" : whatever`, the default value would be `{}`, producing a runtime error.

Now I remember: I modified the default values because the method `void Parameters::SetValue(Parameters Value)` is missing. I can add an extra check in python to set `"value" : 0.0` if it's missing.